### PR TITLE
TrainingStatus performance / robustification

### PR DESCRIPTION
**Pull request by plaursen**
Performance improved a good bit by measuring/optimizing queries in TrainingStatus.

New: Pilots can finish their training programs (!).  Status for pilots in training is on the Pilot editing page (start date set automatically, finish date must be entered). TrainingStatus filters out completed programs (option: show all).

Added activity level icon to TrainingStatus.

Start type shown on Grid.

APRS data sanitized. Still see occasional exceptions from that lib (indexing error).

Fix: don't copy tacho data unless plane has at least one engine.

Still missing translations...

Currently running on dev.startlist.club

### CODE REVIEW

**Comment by plaursen** [comment]
DK translations added

**Comment by plaursen** [comment]
Klar til review :)

File: FlightJournal.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -171,6 +171,8 @@
 <div class="container footer-content">
     <footer>
         <p class="pull-right btn-xs">
+            <a class ="small href="https://www.flaticon.com" title="icons">Icons from Flaticon</a>
```
Comment (janhebnes) [inline FlightJournal.Web/Views/Shared/_Layout.cshtml:174]: kunne vi snige den over til en about side ?

**Review by janhebnes — APPROVED** [review]
Er spændt på at se den live

**Pull request merged**